sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/Samples/Sample04_HookCrudOperations.cs

```diff
@@ -122,6 +122,7 @@ public async Task UpdateHookAsync()
         }
 
         [Test]
+        [Ignore("https://github.com/Azure/azure-sdk-for-net/issues/20677")]
```

Comment:
Issue has been present for more than a week and is not related these changes: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=849790&view=logs&j=68082d65-ebf7-5f3f-494d-d3fd09e80dd6&t=6f6444ca-c101-5878-5224-de85502bb056&l=89